src/avfoundation.cs

```diff
@@ -12250,6 +12392,127 @@ interface AVPlayerItemTrack {
+
+		[NullAllowed, Export ("primaryItem", ArgumentSemantic.Weak)]
+		AVPlayerItem PrimaryItem { get; }
+
+		[Export ("time")]
+		CMTime Time { get; }
+
+		[NullAllowed, Export ("date")]
+		NSDate Date { get; }
+
+		[Export ("templateItems")]
+		AVPlayerItem[] TemplateItems { get; }
+
+		[Export ("interstitialTemplateItems")]
+		AVPlayerItem[] InterstitialTemplateItems { get; }
+
+		[Export ("restrictions")]
+		AVPlayerInterstitialEventRestrictions Restrictions { get; }
+
+		[Export ("resumptionOffset")]
+		CMTime ResumptionOffset { get; }
+	}
+
+	[DisableDefaultCtor]
+	[TV (14,5), Mac (11,3), iOS (14,5)]
+	[BaseType (typeof(NSObject))]
+	interface AVPlayerInterstitialEventObserver
+	{
+		[Static]
+		[Export ("interstitialEventObserverWithPrimaryPlayer:")]
+		AVPlayerInterstitialEventObserver GetInterstitialEventObserver (AVPlayer primaryPlayer);
+
+		[Static]
+		[Export ("playerInterstitialEventObserverWithPrimaryPlayer:")]
+		AVPlayerInterstitialEventObserver GetPlayerInterstitialEventObserver (AVPlayer primaryPlayer);
+
+		[Export ("initWithPrimaryPlayer:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (AVPlayer primaryPlayer);
+
+		[NullAllowed, Export ("primaryPlayer", ArgumentSemantic.Weak)]
+		AVPlayer PrimaryPlayer { get; }
+
+		[NullAllowed, Export ("interstitialPlayer", ArgumentSemantic.Weak)]
+		AVQueuePlayer InterstitialPlayer { get; }
+
+		[Export ("events")]
+		AVPlayerInterstitialEvent[] Events { get; }
+
+		[Export ("interstitialEvents")]
+		AVPlayerInterstitialEvent[] InterstitialEvents { get; }
+
+		[NullAllowed, Export ("currentEvent")]
+		AVPlayerInterstitialEvent CurrentEvent { get; }
+
+		[Field ("AVPlayerInterstitialEventObserverEventsDidChangeNotification")]
+		[Notification]
+		NSString EventsDidChangeNotification { get; }
+
+		[Field ("AVPlayerInterstitialEventObserverCurrentEventDidChangeNotification")]
+		[Notification]
+		NSString CurrentEventDidChangeNotification { get; }
+	}
+
+	[DisableDefaultCtor]
+	[TV (14,5), Mac (11,3), iOS (14,5)]
+	[BaseType (typeof(AVPlayerInterstitialEventObserver))]
+	interface AVPlayerInterstitialEventController
+	{
+		[Static]
+		[Export ("interstitialEventControllerWithPrimaryPlayer:")]
+		AVPlayerInterstitialEventController GetInterstitialEventController (AVPlayer primaryPlayer);
+
+		[Static]
+		[Export ("playerInterstitialEventControllerWithPrimaryPlayer:")]
+		AVPlayerInterstitialEventController GetPlayerInterstitialEventController (AVPlayer primaryPlayer);
+
+		[Export ("initWithPrimaryPlayer:")]
+		IntPtr Constructor (AVPlayer primaryPlayer);
+
+		[NullAllowed, Export ("events", ArgumentSemantic.Copy)]
+		AVPlayerInterstitialEvent[] Events { get; set; }
+
+		[NullAllowed, Export ("interstitialEvents", ArgumentSemantic.Copy)]
+		AVPlayerInterstitialEvent[] InterstitialEvents { get; set; }
+
+		[Export ("cancelCurrentEventWithResumptionOffset:")]
+		void CancelCurrentEventWithResumptionOffset (CMTime resumptionOffset);
```

Comment:
remove `WithResumptionOffset` suffix
that part of the selector is the (first) argument name